psi/cli.py

```diff
@@ -1,11 +1,16 @@
 import sys
-from psi.wsgi import application
+from psi.app import create_app, init_all
+
+
+# Using flask's default `click` command line environment
+application = create_app()
+init_all(application, migrate=False)
```

Comment:
Don't force a migration here. If we want to migrate, manually invoke it from the command line.

Use case: originally I didn't have a database but I wanted to run the command line utilities. Forcing migration means that commands error out if there is no DB available

Reply:
I totally understand the use case you mentioned, the reason why I force the migration is to automatically all necessary step for system start up, so that continues delivery / go live etc can be fully automatised without touch a manual step.

Do you have any suggestion here,  which we could support the use case, still make things fully automatised?

Thanks :)

Reply:
Yes i do :)- it's in the second pull request